tutorials/scripting/instancing_with_signals.rst

```diff
@@ -41,7 +41,7 @@ given velocity:
 
     public partial class Bullet : Area2D
     {
-        public Vector2 Velocity { get; set; } = Vector2.Zero;
+        public Vector2 Velocity { get; set; } = Vector2(1.0, 0.0);
```

Comment:
```suggestion
        public Vector2 Velocity { get; set; } = Vector2.Right;
```
Otherwise it would have to be `new Vector2(1.0, 0.0)` in C#

Reply:
Ah, good catch! I'll take your suggestions.